__tests__/app.test.js

```diff
@@ -0,0 +1,114 @@
+const app = require("../app");
+const db = require("../db/connection");
+const request = require("supertest");
+const testData = require("../db/data/test-data");
+const seed = require("../db/seeds/seed");
+const res = require("express/lib/response");
+
+afterAll(() => {
+  if (db.end) db.end();
```

Comment:
This doesn't need to be in an if statement. You can just return db.end()

Reply:
Please address Rose's feedback.